website/snippets/_run-result.md

```diff
@@ -0,0 +1,2 @@
+- `adapter_response`: Dictionary of metadata returned from the database, which varies by adapter. For example, success `code`, number of `rows_affected`, total `bytes_processed`, and so on. Not populated by tests, as of v0.19.0; we plan to fix in a future release ([dbt#2580](https://github.com/dbt-labs/dbt-core/issues/2580)).
```

Comment:
Can we drop the explicit call-out related to v0.19.0 and replace it with a general statement instead? We closed ([dbt#2580](https://github.com/dbt-labs/dbt-core/issues/2580)) as stale, and I don't think we implemented it.

```suggestion
- `adapter_response`: Dictionary of metadata returned from the database, which varies by adapter. For example, success `code`, number of `rows_affected`, total `bytes_processed`, and so on. Not applicable for [tests](/docs/build/tests).
```

Reply:
omg yes thank you @dbeatty10 ! sugg committed woohoo!